public/app/plugins/datasource/elasticsearch/components/QueryEditor/MetricAggregationsEditor/utils.ts

```diff
@@ -257,6 +257,19 @@ export const metricAggregationConfig: MetricsConfiguration = {
       },
     },
   },
+  rate: {
+    label: 'Rate',
+    xpack: true,
+    versionRange: '>=7.10.0',
+    requiresField: true,
+    isPipelineAgg: false,
+    supportsMissing: false,
+    supportsMultipleBucketPaths: false,
+    hasSettings: true,
+    supportsInlineScript: true,
+    hasMeta: false,
+    defaults: {},
```

Comment:
Would it make sense to provide defaults here? It's still possible to run the query without selecting unit/mode dropdowns but it's not clear what values are used when nothing is selected.

Reply:
@ifrost thanks for the review!

If we add the mode parameter as a default it is sent to ES that returns an error because it requires to have also a field/script parameter when the mode parameter is set explicitly.

For the unit parameter, it uses the same unit that is specified in the date histogram.

So I left it empty because it changes slightly the behavior and I felt that in this way we give more freedom to users to decide how they want the rate aggregation, but I don't have a strong opinion about it.

@simianhacker @Elfo404 what do you think?

Reply:
@Elfo404 I think changing it to `Auto` reads better, but can be updated later, no biggie 👍